docs/features/column-level-lineage.md

```diff
@@ -34,6 +38,69 @@ The transformation type is also displayed for each column, which will help you u
 | Unknown | We have no information about the transformation type. This could be due to a parse error, or other unknown reason. | 
 
 
-## Limitation
+## Impact Radius of a Column
+
+The **right side of the Column-Level Lineage (CLL)** graph represents the **impact radius** of a selected column.
+This view helps you quickly understand what will be affected if that column changes.
+
+### What does the impact radius include?
+
+- **Downstream columns** that directly reference the selected column
+- **Downstream models** that directly depend on the selected column
+- **All indirect downstream columns and models** that transitively depend on it
+
+This helps you evaluate both the direct and downstream effects of a column change, making it easier to understand its overall impact.
+
+
+### Example: Simplified Model Chain
+
+Here are some example models
```

Comment:
Change this:

"Here are some example models"

To be this:

"Given the following models, here's how changes to `stg_orders.status` would affect downstream models:"